src/encoded/static/components/region_search.js

```diff
@@ -1,66 +1,102 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import _ from 'underscore';
 import url from 'url';
-import { BrowserSelector } from './objectutils';
-import { Panel, PanelBody } from '../libs/bootstrap/panel';
-import { FacetList, FilterList, Listing, ResultBrowser } from './search';
-import { FetchedData, Param } from './fetched';
 import * as globals from './globals';
-import _ from 'underscore';
+import { Panel, PanelBody } from '../libs/bootstrap/panel';
+import { FacetList, FilterList } from './search';
 import { SortTablePanel, SortTable } from './sorttable';
 
-
-const regulomeGenomes = [
-    { value: 'GRCh37', display: 'hg19' },
-    { value: 'GRCh38', display: 'GRCh38' },
+const DataTypeStrings = [
+    {
```

Comment:
You need to change `DataTypeStrings` to camel case, along with `ExampleEntries`. Didn’t know until now AirBnB has a rule on that (which ESLint didn’t pick up): https://github.com/airbnb/javascript#naming--camelCase